File: Array/412. Fizz Buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> dataArray;
        string t="";
        for( int i = 1; i <= n; i++ ) {
            if(i%3==0 && i%5==0)
            {
                dataArray.push_back("FizzBuzz");
            }
            else if(i%3==0)
            {
                dataArray.push_back("Fizz");
            }
            else if(i%5==0)
            {
                dataArray.push_back("Buzz");
            }
            else
            {
                t = to_string(i);
                                
                dataArray.push_back( t );                

            }
            
        }
        return dataArray;
    }
};
// Runtime: 8 ms
// Memory Usage: 7.8 MB

File: Array/412. Fizz Buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        new_list = list()
        for j in range(n):
            i=j+1
            if i%3==0:
                if i%5==0:
                    new_list.append("FizzBuzz")
                else:
                    new_list.append("Fizz")
            elif i%5==0:
                new_list.append("Buzz")
            else:
                new_list.append(str(i))
        return new_list
